modules/virt-understanding-live-migration.adoc

```diff
@@ -1,20 +1,19 @@
 // Module included in the following assemblies:
 //
 // * virt/live_migration/virt-live-migration.adoc
+// * virt/upgrading-virt.adoc
 
 [id="virt-understanding-live-migration_{context}"]
 = Understanding live migration
 
-Live migration is the process of moving a running virtual machine instance (VMI) to
-another node in the cluster without interrupting the virtual workload or access.
-If a VMI uses the `LiveMigrate` eviction strategy, it automatically migrates
-when the node that the VMI runs on is placed into maintenance mode. You can also
-manually start live migration by selecting a VMI to migrate.
+Live migration is the process of moving a running virtual machine instance (VMI) to another node in the cluster without interrupting the virtual workload or access. If a VMI uses the `LiveMigrate` eviction strategy, it automatically migrates when the node that the VMI runs on is placed into maintenance mode. You can also manually start live migration by selecting a VMI to migrate.
```

Comment:
There is already a section for "Live migration". I would put this content under that section and give only reference. I could make this page more clear as well.